utils/httpops.go

```diff
@@ -0,0 +1,45 @@
+package utils
+
+import (
+	"net/http"
+	"strings"
+)
+
+func RunHTTPGet(url string) (*http.Response, error) {
+	client := &http.Client{}
```

Comment:
Please set a reasonable timeout. Golang does the annoying thing of having the default value be forever